### DISCUSSION TITLE
Native blameables

### DISCUSSION THREAD

**Discussion by denny-bee** [laravel/framework#51607] [Ideas]
Most of the applications today need blames. Timestamps are included by default in the framework. In my opinion blameables should be too.

**Comment by ghost** [comment]
Hi. Aren't the git blames enough?

**Reply by denny-bee** [reply]
I meant a field named created, updated and deleted_by_user_id (only if softdeletes are active for the delete blaming field).
So they are set when the matching timestamp fields like created_at are set.
In addition to this implemented relations would be awesome too like createdByUser.
This could be done with the implantation of an interface, which is used to recognize if a model uses this feature or by a trait.
I think this is a feature which most developers need.
